yt/java/ytsaurus-client/src/main/java/tech/ytsaurus/client/ImmutableTransactionalClient.java

```diff
@@ -35,6 +36,8 @@ default <T> CompletableFuture<List<T>> lookupRows(
         return lookupRows(request.build(), serializer);
     }
 
+    CompletableFuture<List<UnversionedRowset>> multiLookupRows(AbstractMultiLookupRequest<?, ?> request);
```

Comment:
Could you please add and implement this method in [SyncTransactionalClient](https://github.com/ytsaurus/ytsaurus/blob/main/yt/java/ytsaurus-client/src/main/java/tech/ytsaurus/client/sync/SyncTransactionalClient.java#L75) as well?